### PR TITLE
Release 2.11

### LINKED ISSUE #12499: vue-2 skeleton creates a broken project
linux ubuntu 20
meteor 2.10

### To Reproduce

Run the command

```
meteor create --vue-2 example
```

### expected behaviour

The example project that's created should work as designed (i.e. a single page that shows a counter and the list of urls that can be added to) without errors in the console.

### actual behaviour

The project builds but the list of urls is not shown because the subscription fails, creating an error in the console.

### Notes

* this behaviour has been seen vue 2.7 was released (Jul 22)
* the fix is to pin the vue package dependency to 2.6 (see https://github.com/meteor-vue/vue-meteor-tracker/issues/75)

### PR DESCRIPTION
**Pull request by Grubba27**
# Still in development
Current version: ``2.11.0``
To use it, please do a 
```bash
meteor update --release  2.11.0
```

## Highlights

* MongoDB Server 6.x Support
* Embedded Mongo now uses MongoDB 6.0.3
* #12462 by [radekmie](https://github.com/radekmie)  
* #12469 by [ebroder](https://github.com/ebroder)
* #12465 by [Firfi](https://github.com/Firfi) 
* #12480 by [Torgen](https://github.com/Torgen) 
* #12456 by [StorytellerCZ](https://github.com/StorytellerCZ) 
* #12478 by [softwarecreations](https://github.com/softwarecreations) .

Preview docs: https://deploy-preview-12470--meteor-docs.netlify.app/

## Maintainers checklist
TODO

closes #12499

### CODE REVIEW

**Comment by zodern** [comment]
Would it be possible to fix https://github.com/meteor/meteor/issues/12301 in 2.11? It would be nice to have it fixed before Meteor 3 so apps that need to support old browsers can update.

**Comment by hexsprite** [comment]
Just running the dev server I'm getting occasional errors like this (2.11-rc.0)

```
Exception in setInterval callback: PoolClearedOnNetworkError: Connection to 127.0.0.1:4001 interrupted due to server monitor timeout
    at ConnectionPool.interruptInUseConnections (/Users/jbb/.meteor/packages/npm-mongo/.4.14.0-rc2110.0.1ywynt2.wqmh++os+web.browser+web.browser.legacy+web.cordova/npm/node_modules/mongodb/lib/cmap/connection_pool.js:264:36)
    at /Users/jbb/.meteor/packages/npm-mongo/.4.14.0-rc2110.0.1ywynt2.wqmh++os+web.browser+web.browser.legacy+web.cordova/npm/node_modules/mongodb/lib/cmap/connection_pool.js:251:41
    at processTicksAndRejections (internal/process/task_queues.js:77:11)
    at runNextTicks (internal/process/task_queues.js:64:3)
    at listOnTimeout (internal/timers.js:526:9)
    at processTimers (internal/timers.js:500:7) {
  address: '127.0.0.1:4001',
  [Symbol(errorLabels)]: Set(1) { 'RetryableWriteError' }
}
```

**Comment by hexsprite** [comment]
Also I tried upgrade to 2.11-rc.1 but got this message:

```
This project uses Meteor 2.11.0-rc.1, which isn't available on this platform. To work with this app on all supported platforms, use meteor update --release METEOR@2.10.0 to pin this app to the newest compatible release.
```

I'm on an M1 Macbook Pro...

**Comment by Grubba27** [comment]
> Just running the dev server I'm getting occasional errors like this (2.11-rc.0)
 
That can be an issue. I will take a look.

> Also I tried upgrade to 2.11-rc.1 but got this message:
This project uses Meteor 2.11.0-rc.1, which isn't available on this platform. To work with this app on all supported platforms, use meteor update --release METEOR@2.10.0 to pin this app to the newest compatible release.
I'm on an M1 Macbook Pro...

Our Jenkins CI was off, and this week, I was working full-time on fibers, so I could not check on this. 
The good news is that from what I can see while I'm writing the message Jenkins is running and I will be able to fully make this release

**Comment by Grubba27** [comment]
Hey, @hexsprite I got it to work 😄 

Screenshot 2023-02-17 at 18 42 18

**Comment by hexsprite** [comment]
> Exception in setInterval callback: PoolClearedOnNetworkError: Connection to 127.0.0.1:4001 interrupted due to server monitor timeout

It might be related to the machine going to sleep while AFK. Not critical and it still continues to work after I get back without needing to be restarted...

**Comment by Grubba27** [comment]
@ebroder heyy, the "typescript template works" still fails :/ not sure why. If this does not work I will create a app in tests/apps, I guess this way things will work out of the box

**Comment by Grubba27** [comment]
Oh, got it, running it from a local web as this one https://github.com/meteor/meteor/blob/a69f2e573e8cefe5e6c52b7bf2d86f2f00bce3a1/tools/tests/regressions.js#L10
would work? 
I'm working on making another rc

**Comment by ebroder** [comment]
I'm not sure it's possible to fix this on the Meteor side right now. I think this has to be fixed from `zodern:types`. So maybe we should just skip the test for now until that gets fixed.

File: packages/mongo/mongo.d.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export namespace Mongo {
          * The server connection that will manage this collection. Uses the default connection if not specified. Pass the return value of calling `DDP.connect` to specify a different
          * server. Pass `null` to specify no connection. Unmanaged (`name` is null) collections cannot specify a connection.
          */
-        connection?: Object | null | undefined;
+        connection?: DDP.DDPStatic | null | undefined;
```
Comment (Grubba27) [inline packages/mongo/mongo.d.ts:63]: Here was missing a `import { DDP } from 'meteor/ddp';` will be added before the next rc

File: docs/generators/changelog/versions/2.11.md [hunk 2]
```diff
@@ -25,6 +25,7 @@
 * Add type definitions for async methods from Meteor 2.9 by [GH ebroder] [PR #12507]
 * TypeScript skeleton fixes by [GH ebroder] [PR #12506]
 * Fix TypeScript type dependencies for mongo, webapp, and underscore by [GH ebroder] [PR #12505]
+* Improve specificity of types previously declared as "Object" by [GH ebroder] [PR #12520]
```
Comment (StorytellerCZ) [inline docs/generators/changelog/versions/2.11.md:28]: Node update is missing here.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: docs/generators/changelog/versions/2.11.md (hunk 2 above)
Comment (Grubba27) [reply docs/generators/changelog/versions/2.11.md:28]: Nice! Done. 
I will make recheck again when I start working on the official release for this one
(Wished that gh had a page showing wich prs were merged in branch)

File: docs/generators/changelog/versions/2.11.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
+## v2.11.0, 2023-02-XX
```
Comment (StorytellerCZ) [inline docs/generators/changelog/versions/2.11.md:1]: Don't forget to change the date here.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: docs/generators/changelog/versions/2.11.md (hunk 3 above)
Comment (Grubba27) [reply docs/generators/changelog/versions/2.11.md]: got it! thanks again @StorytellerCZ :)

**Review by StorytellerCZ — APPROVED** [review]

**Comment by filipenevola** [comment]
@Grubba27 I think this is wrong in the PR description: 

`meteor update --release=2.11.0`

Maybe this was the root cause of this [user](https://forums.meteor.com/t/macos-13-2-1-cant-update-to-release-2-11-0/59634) trying to update to 2.11.0

**Pull request merged**